en/simulation/gazebo.md

```diff
@@ -175,74 +180,86 @@ In addition to the existing cmake targets that run `sitl_run.sh` with parameters
 
 This approach significantly reduces the debug cycle time because simulator (e.g. gazebo) is always running in background and you only re-run the px4 process which is very light.
 
+
 ## Video Streaming
-PX4 SITL for Gazebo supports UDP Video Streaming. The video from gazebo camera sensor, attached to a model, is streamed over UDP port 5600 and can be viewed remotely. The video is streamed by a gstreamer pipeline
 
-### Prerequisite
-* Install Gstreamer 1.0
+PX4 SITL for Gazebo supports UDP video streaming from a Gazebo camera sensor attached to a vehicle model. You can connect to this stream from *QGroundControl* (on UDP port 5600) and view video of the Gazebo environment from the simulated vehicle - just as you would from a real camera. The video is streamed using a *gstreamer* pipeline.
+
+> **Note** Video streaming from Gazebo and the Gazebo widget to turn streaming on/off are not enabled by default. This article explains how to enable them. In the near future we expect these features to be enabled by default.
+
+### Prerequisites
+
+Install *Gstreamer 1.0*:
 ```
-sudo apt-get install gstreamer1.0*
+sudo apt-get install gstreamer1.0* 
```

Comment:
@lbegani @dagar 
1. Is the "*" at the end required? 
2. This fails on our supported Linux platform (Ubuntu LTS 16.04) with the following error:
```
...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 gstreamer1.0-hybris:i386 : Depends: libmedia1:i386 but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```
Any thoughts on right way to fix that (I will update our build scripts to include the "right" lines by default.

Reply:
I haven't tried it myself but perhaps we can take reference from https://github.com/mavlink/qgroundcontrol/tree/master/src/VideoStreaming and replace it with the following steps - 
-> list=$(apt-cache --names-only search ^gstreamer1.0-* | awk '{ print $1 }' | grep -v gstreamer1.0-hybris)
-> sudo apt-get install $list

Reply:
Cool! I tried `sudo apt-get install $(apt-cache --names-only search ^gstreamer1.0-* | awk '{ print $1 }' | grep -v gstreamer1.0-hybris) -y` and it works (though I don't really understand it).

@dagar Thoughts on adding that line to the docs, build scripts (and presumably docker?).

Reply:
Hmm, I'm really prefer if we can figure out the actual package requirements. Maybe you can figure out what's actually being used by running it under ltrace?

Reply:
@dagar For now I have put in that line above - it works and I need to fix the ugly rendering fast. I will merge shortly. 
I'll hold off updating any scripts etc. To my mind that doesn't get done until the plugin is on by default, at which point it has to be done. At that point we should consider what the dependencies are.